FunctionalTableData/TableView/FunctionalTableData.swift

```diff
@@ -450,6 +450,16 @@ public class FunctionalTableData: NSObject {
 		aTableView.scrollToRow(at: indexPath, at: scrollPosition, animated: animated)
 	}
 	
+	/// - Parameter point: The point in the collection view’s bounds that you want to test.
```

Comment:
Nitpick: `collection view's` -> `table view's`

Reply:
This doesn’t look like it was fixed 🤔

Reply:
Argh, you're right. That was sloppy of me.